### DISCUSSION TITLE
failed to start starknet

### DISCUSSION THREAD

**Discussion by humberto701** [eqlabs/pathfinder#1016] [Q&A] [answered]
-- Logs begin at Mon 2020-05-18 09:51:57 CEST. --
Apr 07 00:08:16 vm11629.stark-industries.solutions systemd[1]: Started StarkNet.
Apr 07 00:08:16 vm11629.stark-industries.solutions bash[94103]: error: The argument '--ethereum.url <HTTP(s) URL>' requires a value but none was supplied
Apr 07 00:08:16 vm11629.stark-industries.solutions bash[94103]: For more information try --help
Apr 07 00:08:16 vm11629.stark-industries.solutions systemd[1]: starknetd.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Apr 07 00:08:16 vm11629.stark-industries.solutions systemd[1]: starknetd.service: Failed with result 'exit-code'.
Apr 07 00:08:16 vm11629.stark-industries.solutions systemd[1]: starknetd.service: Scheduled restart job, restart counter is at 5.
Apr 07 00:08:16 vm11629.stark-industries.solutions systemd[1]: Stopped StarkNet.
Apr 07 00:08:16 vm11629.stark-industries.solutions systemd[1]: starknetd.service: Start request repeated too quickly.
Apr 07 00:08:16 vm11629.stark-industries.solutions systemd[1]: starknetd.service: Failed with result 'exit-code'.
Apr 07 00:08:16 vm11629.stark-industries.solutions systemd[1]: Failed to start StarkNet.

**Comment by sergey-melnychuk** [comment, marked as answer]
> `The argument '--ethereum.url <HTTP(s) URL>' requires a value but none was supplied`. 

The `ethereum.url` must be provided. See https://www.newton.so/view/63a0659a407c7621270c0204.
